lib/autoprefixer.js

```diff
@@ -6,6 +6,7 @@ let Browsers = require('./browsers')
 let Prefixes = require('./prefixes')
 let data = require('../data/prefixes')
 let info = require('./info')
+let { Cache } = require('./utils')
```

Comment:
Can you put it to the first line to order imports by name length `{ Cache } > info`